addons/medical/functions/fnc_treatmentOrthopedicCast.sqf

```diff
@@ -0,0 +1,43 @@
+/*
+ * Author: Orbis2358
+ * Patient Cast Treatment callback
+ *
+ * Arguments:
+ * 0: The medic <OBJECT>
+ * 1: The patient <OBJECT>
+ * 2: SelectionName <STRING>
+ * 3: Treatment classname <STRING>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [medic, patient, "SelectionName", "cast"] call ace_medical_fnc_treatmentOrthopedicCast
+ *
+ * Public: Yes
+ */
+
+#include "script_component.hpp"
+
+params ["_caller", "_target", "_selectionName", "_className", "_items"];
+
+if (count _items == 0) exitWith {false};
+
+private _part = [_selectionName] call FUNC(selectionNameToNumber);
+if (_part == 0 || _part == 1) exitWith {false};
+
+// private _cast = _target getVariable [QGVAR(orthopedicCast), [[0, 0], [0, 0], [0, 0], [0, 0], [0, 0], [0, 0]]];
+// if ((_cast select _part select 0) > 0) exitWith {false};
+
+private _removeItem = _items select 0;
+if (local _target) then {
```

Comment:
doesn't CBA targetEvent do that internally already? not sure

Reply:
Not sure about that either. I just referenced from other functions in ACE Medical for that. Maybe for optimization?

Reply:
not worth fixing the others. Medical rewrite will do that anyway